.github/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+# Description
+
+Addresses # (issue)
+
+Please include a summary of the change and which issue is fixed. Please also
+include relevant motivation and context. List any dependencies that are required
+for this change.  Please see ``CONTRIBUTING.md`` for more guidelines.
+
+# Type of change
+
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality or documentation)
+- Breaking change (fix or feature that would cause existing functionality
+      to not work as expected)
+- This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide
+instructions so we can reproduce the problem and the solution. Including images
+or test files is frequently very useful.  Please also list any relevant details
+for your test configuration.
+
+- Test A
+
+```
+Test B
+```
+
+## Test Configuration
+* Operating system: Hal
+* Version number: Python 3.X
+* Any details about your local setup that are relevant
+
+# Checklist:
+
+- [ ] Make sure you are merging into the ``develop`` (not ``main``) branch
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have linted the files updated in this pull request
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
Should we still have a downstream check for a pysat* package?
Personally it seems to me that the check should be for upstream.

Reply:
Well, we can't break anything upstream, so I don't think we need a check there.  While at the moment there isn't anything downstream from here, it's possible that analysis packages could rely on instrument methods in the future.  So that is why I am keeping it here :)